cli/index.js

```diff
@@ -171,95 +171,127 @@ async function createIssues() {
     }, {});
     issues = Object.values(reduced);
 
-    if (conf.batch) {
-        // filter down to the package that was picked
-        issues = (await getBatchProps(issues)).issues;
-    }
-
-    if (conf.autoGenerate) {
+    let continuePrompting = true;
+    while (continuePrompting) {
```

Comment:
This diff is terrible, but I just moved this whole block into a `while` loop.

Reply:
I clicked "ignore whitespace" in the diff and it looked a little better

Reply:
TIL!